packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/GeneratedCameraXLibrary.java

```diff
@@ -1,7 +1,7 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-// Autogenerated from Pigeon (v9.1.1), do not edit directly.
+// Autogenerated from Pigeon (v9.2.4), do not edit directly.
```

Comment:
All of our generated files besides java already use that naming convention. I think there was a reason we didn't for java, but you'd have to ask @stuartmorgan , since he set that up.

Reply:
We can't 🙁 It's apparently a build error for the Java file containing `public class SomeClass` to be named anything other than `SomeClass.java`.

Reply:
Oh well darn